purchase_order_report_selection/security/ir.model.access.csv

```diff
@@ -0,0 +1,2 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_purchase_report_user,purchase.report.field,model_purchase_report_field,purchase.group_purchase_user,1,1,1,1
```

Comment:
Are you sure these can be configured by regular user not by project manager?
This is not an issue, just asking.

Reply:
I thought that the user is an internal user with limited rights, so I gave him the rights.
Although it could be done so that the manager adds the necessary fields, and the user selects from these fields the necessary ones for himself.